init.lua

```diff
@@ -51,7 +51,7 @@ local function register_liquid_wood(source, itemname, inventory_image, name, gro
 					return
 				end
 
-				local above_ndef = node and minetest.registered_nodes[node.name]
+				local above_ndef = nnode and minetest.registered_nodes[nnode.name]
```

Comment:
the test for nnode can be skipped in line 54 as line 50 already does that ==>
`` local above_ndef = core.registered_nodes[nnode.name]``

Reply:
being nit-picky: why not call it ``above_node`` instead of ``nnode``?

Reply:
I just went with the variable that was already there. The MTG bucket mod redefines the `node` variable instead of declaring a new one.

Reply:
I agree with your other comment. It's definitely redundant to have lines 50 through 53 and the nnode check on 54. To be consistent with the MTG bucket mod, I would recommend removing lines 50 through 53.

Reply:
I'd rather have easy to read and debug code than code that is duplicating bad habits for consistancy's sake.

If the variable had been named better in the MTG code, there wouldn't have been this confusion in the first place.

Keeping the if-block starting at line 50 gives a good debug point and shortens the ``above_node_def`` definition line, allowing again for more expressive variable name usage.

Just my two cents ;) no obligation.